controllers/members.js

```diff
@@ -32,7 +32,7 @@ const getMembers = async (req, res) => {
 const getIdleMembers = async (req, res) => {
   try {
     let allMemberUsernames = await memberQuery.fetchMembers()
-    allMemberUsernames = allMemberUsernames.map(member => member.username)
+    allMemberUsernames = allMemberUsernames.oldMembers.map(member => member.username)
```

Comment:
This is to find the idle members only from old members, right? Just confirming.

Reply:
Basically the structure of the returned json has been changed from allMembers = [{},{},{},...] to allMembers = {newMembers: [{},{},{},...], oldMembers: [{},{},{},...]}.

So I believe, we are considering idle members from oldMembers not from newMembers. That is my take. 

We should confirm this assumption with AO. @sumitd94 

In fact, Sumit suggested this json structure change in his previous review.

Reply:
I'm also assuming the same. @sumitd94 Please confirm.

Reply:
Yes, idle members should be returned from the list of `old-members` (here by old-members we are referring to all those users who has `isMember` set to `true`